### DISCUSSION TITLE
Entanglement calculation in biorthogonal basis

### DISCUSSION THREAD

**Discussion by Sreema1996** [QuSpin/QuSpin#675] [General]
Hi Developers,

I am trying to calculate the entanglement entropy for a non-Hermitian model in biorthogonal basis, where the density matrix of the full system is defined as follows \rho_{n}=\ket{n_{R}}\bra{n_{L}}, where \ket{n_{R}} and \bra{n_{L}} are the nth right and left eigenstate. Is there any way to implement this using the usual QuSpin's entanglement entropy calculation method?

Thanks in advance!

**Comment by mgbukov** [comment]
quspin uses the z-basis as a computational basis for the entanglement
entropy calculations.

But if you write a function that correctly computes Sent in an arbitrary
basis, share it with the rest of the community please :)

On Wed, Apr 24, 2024 at 8:30 AM Sreema1996 ***@***.***> wrote:

> Hi Developers,
>
> I am trying to calculate the entanglement entropy for a non-Hermitian
> model in biorthogonal basis, where the density matrix of the full system is
> defined as follows \rho_{n}=\ket{n_{R}}\bra{n_{L}}, where \ket{n_{R}} and
> \bra{n_{L}} are the nth right and left eigenstate. Is there any way to
> implement this using the usual QuSpin's entanglement entropy calculation
> method?
>
> Thanks in advance!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/QuSpin/QuSpin/discussions/675>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADHSUBUKRQ7N2HFHVLAORYLY65GRXAVCNFSM6AAAAABGWIUCIOVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGU2TEMRWGI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Marín Bukov, PhD
Research Group Leader
Max Planck Institute for the Physics of Complex Systems
Noethnitzer Str. 38
Dresden 01187, Germany

url: https://www.pks.mpg.de/nqd
<http://quantum-dynamics.phys.uni-sofia.bg/>
